OloEngine/src/OloEngine/Scripting/Lua/LuaScriptGlue.cpp

```
@@ -36,20 +38,394 @@
 #include "OloEngine/Physics3D/JoltScene.h"
 #include "OloEngine/Audio/AudioEvents/AudioPlayback.h"
 #include "OloEngine/Audio/AudioEvents/CommandID.h"
+#include "OloEngine/Scene/Streaming/SceneStreamer.h"
+
+#include "box2d/box2d.h"
 
 #include <algorithm>
```

Cppcheck (2.20.0)

[information] Include file

(missingIncludeSystem)